examples/electricity/two_resistors.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+from sympy import solve
+from symplyphysics import Symbol, units
+from symplyphysics.laws.electricity import current_is_proportional_to_voltage as ohm_law
+from symplyphysics.laws.electricity.circuits import resistivity_of_parallel_resistors as parallel_resistors
+from symplyphysics.laws.electricity.circuits import resistivity_of_serial_resistors as serial_resistors
+
+# Two resistors are connected across a 12 V battery with internal resistance of 1 Ohm.
+# When they are connected in parallel, the current in the circuit is 4 A.
+# When they are connected in series, the current in the circuit is 1 A.
+# Find the resistance of said resistors.
+
+R1 = Symbol("R1", units.impedance)
```

Comment:
Use sympy symbols and remove dimensions.

Reply:
Okay, other examples use `Symbol` though.

Reply:
There are two different `Symbol`s. One from sympy and another from symplyphysics. Use the sympy's one.